pages/content/ch2/implications.mdx

```diff
@@ -7,4 +7,36 @@ import { Callout,Tabs } from 'nextra/components'
 
 # Implications
 
-$\to$, $\implies$, $\overset{\cdot}{\implies}$, $\models$, $\vdash$
+There is a multitude of meanings when it comes to the use of the word "implication". There is a good detailed explanation in the script.
+To provide an overview consider the following:
+
+
+## Implications between Mathematical statments
+
+The arrow "$\implies$" can be used between mathematical statements to form a new mathematical statement. For example, let $S$ and $T$ be mathematical statements. Then we can form the mathematical statement $S \implies T$. This simply means that **if** $S$ is true, **then** $T$ must also be true. It does **not** mean that **because** $S$ is true, $T$ is also true.
```

Comment:
Ja die Anwendung ist korrekt... aber das resultierende statement is falsch... vielleicht könnte man das noch irgendwo expilizit hinschreiben... oder hab ich irgendwo geschrieben, dass es keine korrekte anwendung ist?

Reply:
ah ich glaub ich weiss was du meinst... ich versuche es besser zu formulieren

Reply:
"This simply means that **if** $S$ is true, **then** $T$ must also be true."

vllt zu sowas:

"This simply means that **if** $S$ is true, **then** $T$ must also be true (otherwise the new statement is false)."

Reply:
wäre so okay?

Reply:
ja inhaltlich jetzt 100% korrekt, aber vllt noch etwas spezifischer schreiben dass das nur ein statement ist (welches auch falsch sein kann). viele haben das (in meiner uebung) mit den statements nicht so ganz verstanden, da ist es vllt hilfreich sehr detalliert zu sein.

d.h. ich wuerde vorschlagen:

that ***if** $S$ is true, **then** $T$ is also true*

zu (mit anfuehrungszeichen damit klar ist was teil des statements ist):

"that ***if** $S$ is true, **then** $T$ is also true*" (which is a statement that can be false).